doc/source/diagnostics/extended_show_grants.rst

```diff
@@ -0,0 +1,72 @@
+.. _extended_show_grants:
+
+==========================
+ Extended ``SHOW GRANTS``
+==========================
+
+Prior to |Percona Server| :rn:`5.5.42-37.1`, ``SHOW GRANTS`` displays only the privileges granted explicitly to the named account. Other privileges might be available to the account, but they are not displayed. For example, if an anonymous account exists, the named account might be able to use its privileges, but ``SHOW GRANTS`` will not display them. In :rn:`5.5.42-37.1` ``SHOW GRANTS`` command was extended to display all the privileges which granted to the account.
```

Comment:
... ``SHOW GRANTS`` command was extended to display all the effectively available privileges to the account.